README.md

```diff
@@ -184,6 +184,15 @@ Check out about Navigation: [https://grand-ui.netlify.app/getting-started/docume
 
 <hr>
 
+### Modal
+
+Modal can be used for different purposes which is kind of a popup boxes that appear on screen. These modals usually require some actions to cancel or handle them. 
+
+
+Check out about Modal: [https://grand-ui.netlify.app/getting-started/documentation.html#modal](https://grand-ui.netlify.app/getting-started/documentation.html#modal)
```

Comment:
Don't do this!
This is basically repeating itself,
either put the link directly, or
do this : `[ Modal ]( https://grand-ui.netlify.app/getting-started/documentation.html#modal )`
output : [ Modal ]( https://grand-ui.netlify.app/getting-started/documentation.html#modal )

Reply:
Yeah I understand what you are saying but I did this on purpose taking into consideration that I want user to view link as it is. Also if user has a habit of copy pasting links this would also help in that manner I  have kept it this way.

Reply:
okay! for that you can put the link directly not needed to wrap it
like this: `https://grand-ui.netlify.app/getting-started/documentation.html#modal`
output: https://grand-ui.netlify.app/getting-started/documentation.html#modal